docs/literate/src/files/subcell_shock_capturing.jl

```diff
@@ -0,0 +1,273 @@
+# SSPRK time integration methods with passed stage callbacks and a Trixi-intern `Trixi.solve(...)`
+# routine.
+stage_callbacks = (SubcellLimiterIDPCorrection(),)
+
+sol = Trixi.solve(ode, Trixi.SimpleSSPRK33(stage_callbacks = stage_callbacks);
+                  dt = 1.0, # solve needs some value here but it will be overwritten by the stepsize_callback
+                  callback = callbacks);
+summary_callback() # print the timer summary
+
+
+# ## Visualization
+# As for a standard simulation in Trixi.jl, it is possible to visualize the solution using the
+# `plot` routine from Plots.jl.
+using Plots
+plot(sol)
+
+# To get an additional look at the amount of limiting that is used, you can use the visualization
+# approach using the [`SaveSolutionCallback`](@ref), [`Trixi2Vtk`](https://github.com/trixi-framework/Trixi2Vtk.jl)
+# and [ParaView](https://www.paraview.org/download/). More details about this procedure
+# can be found in the [visualization documentation](@ref visualization).
+# Unfortunately, the support for subcell limiting data is not yet merge into the main branch
+# of Trixi2Vtk but lies in the branch `bennibolm/node-variables`.
+#-
+# With that implementation and the standard procedure used for Trixi2Vtk you get the following
+# dropdown menu in ParaView.
+#-
+# ![ParaView_Dropdownmenu](https://github.com/trixi-framework/Trixi.jl/assets/74359358/70d15f6a-059b-4349-8291-68d9ab3af43e)
+
+# The resulting visualization of the density and the limiting parameter then looks like this.
+# ![blast_wave_paraview](https://github.com/trixi-framework/Trixi.jl/assets/74359358/e5808bed-c8ab-43bf-af7a-050fe43dd630)
+
+# You can see that the limiting coefficient does not lie in the interval [0,1], what actually was
+# expected due to its calculation.
+# TODO: Did I write something about this calculation?
+# This is due to the reconstruction functionality which is defaultly enabled in Trixi2Vtk.
+# You can disabled it with `reinterpolate=false` within the call of `trixi2vtk(...)` and get the
+# following visualization.
+# ![blast_wave_paraview_reinterpolate=false](https://github.com/trixi-framework/Trixi.jl/assets/74359358/39274f18-0064-469c-b4da-bac4b843e116)
+
+
+# ## Bounds checking
+# Subcell limiting is based on the fulfillment of target bounds - either global or local.
+# Although the implementation is correct there are some cases where these bounds are not met.
+# On the one hand, the deviations could be in machine precision which is not problematic.
+# On the other hand, the target bound can be exceeded by larger deviations. Mostly, this is due
+# to too large time step sizes and can be easily fixed by reducing the CFL number.
+# Rarely, the larger deviations come from the use of specific boundary condition of source terms.
+#-
+# In all this cases the exceeded bounds are not too problematic. Though, it is reasonable to
+# monitor them. Because of that Trixi.jl supports a bounds checking routine implemented using
```

Comment:
Not always true
```suggestion
# In many cases, it is reasonable to monitor the bounds deviations.
# Because of that, Trixi.jl supports a bounds checking routine implemented using
```